app/routes.js

```diff
@@ -1,13 +1,19 @@
 // const controller = require('./controllers/controller');
 const { getAlbums, getAlbumPhotos } = require('./controllers/albums'),
   { healthCheck } = require('./controllers/healthCheck'),
-  { signUp, signIn } = require('./controllers/users'),
-  { checkValidEmailAndPassword, checkValidName } = require('../app/middlewares/users');
+  { signUp, signIn, listUsers } = require('./controllers/users'),
+  {
+    checkValidEmailAndPassword,
+    checkValidName,
+    checkValidOffsetAndLimit,
+    checkNotNullToken
+  } = require('../app/middlewares/users');
```

Comment:
These are starting to pile up. Maybe just import user middleware and do `user.checkValidEmailAndPassword` and such? 
Also, the name of the file is fine but i'd move it inside a validations folder, inside the middleware folder, so it is clear it's for user validations, and not some other user logic.